reflex/event.py

```diff
@@ -1069,13 +1144,18 @@ def parse_args_spec(arg_spec: ArgsSpec):
     )
 
 
-def check_fn_match_arg_spec(fn: Callable, arg_spec: ArgsSpec) -> List[Var]:
+def check_fn_match_arg_spec(
+    fn: Callable,
+    arg_spec: ArgsSpec,
+    key: Optional[str] = None,
```

Comment:
`key` is not used inside this particular method, should we remove the args or did you mean to use it somewhere?